### PR TITLE
Add readme with example of utilisation and installation

### PR DESCRIPTION
**Pull request by Ydrasil**

### CODE REVIEW

**Comment by Ydrasil** [comment]
@vinyll

File: readme.md [hunk 1]
```diff
@@ -1,3 +1,40 @@
 # TCI Translation Library
 
 Translate the TCI interface from CSV translation files.
+
+## Example
+
+Tci-i18n provide translation from CSV translation files.
```
Comment (vinyll) [inline readme.md]: from CSV ~translation~ files.

File: readme.md [hunk 2]
```diff
@@ -1,3 +1,40 @@
...
+
+### CSV files format
+
+CSV files can have this format
```
Comment (vinyll) [inline readme.md]: > … and should be UTF-8 encoded

File: readme.md [hunk 3]
```diff
@@ -1,3 +1,40 @@
...
+from tcii18n import Translator
+
+translator = Translator('my_file.csv')
+translator.translate('My string')
+```
+
+## Installation
+
+```
+python3 -m venv venv
```
Comment (vinyll) [inline readme.md]: We don't need any virtualenv here as there's no specific installation

File: readme.md [hunk 4]
```diff
@@ -1,3 +1,40 @@
...
+source venv/bin/activate
+```
+
+## Running tests
+
+```
+source venv/bin/activate
```
Comment (vinyll) [inline readme.md]: Same for tests

**Review by vinyll — CHANGES_REQUESTED** [review]

**Comment by Ydrasil** [comment]
@vinyll

File: readme.md [hunk 5]
```diff
@@ -1,3 +1,26 @@
 # TCI Translation Library
 
 Translate the TCI interface from CSV translation files.
+
+## Example
+
+Tci-i18n provide translation from CSV files.
```
Comment (vinyll) [inline readme.md]: TCI-i18n _provides_

File: readme.md [hunk 6]
```diff
@@ -1,3 +1,26 @@
...
+
+### CSV files format
+
+CSV files can have this format and and should be UTF-8 encoded
```
Comment (vinyll) [inline readme.md]: missing the final point "."

File: readme.md [hunk 7]
```diff
@@ -1,3 +1,26 @@
...
+
+```
+TCI title,Titre ITC
+"yes, or no","oui, ou non"
+Welcome %(user)s!,Bienvenue %(user)s !
+```
+
+### Utilisation
```
Comment (vinyll) [inline readme.md]: `## Usage`

**Review by vinyll — CHANGES_REQUESTED** [review]

**Review by vinyll — APPROVED** [review]

**Pull request merged**